polars/polars-core/src/chunked_array/ops/repeat_by.rs

```diff
@@ -6,11 +6,30 @@ use crate::prelude::*;
 
 type LargeListArray = ListArray<i64>;
 
+fn assert_lengths(length_srs: usize, length_by: usize) -> PolarsResult<()> {
```

Comment:
Shall we call this `check_lengths`? Assert seems to me we will panic on invalid inputs, which we don't/shouldn't.

Reply:
done